api/placer.py

```diff
@@ -82,13 +82,15 @@ def save_file(self, field, info):
         """
 
         # Save file
-        files.move_form_file_field_into_cas(field)
+        if field is not None:
+            files.move_form_file_field_into_cas(field)
```

Comment:
Yes renaming it and splitting could be the right thing, but I didn't want to modify/break the behavior of other placers.

For analyses, for example, I needed a different update logic for the file info, so I could reuse only the `move file` part of the save file.